### ISSUE TITLE
Can I use it on React-Native?

### ISSUE THREAD

**Issue by jannomeister** [signavio/react-mentions#413]

**Comment by alex-mironov** [comment]
You cannot. You can try those libraries, but they're poorly maintained, and don't look solid 
https://github.com/harshq/react-native-mentions
https://github.com/mrazadar/react-native-mentions-editor

**Comment by juddey** [comment]
They're not solid. Can tell you from experience!

**Comment by walidvb** [comment]
i'm wondering if it is actually possible to port this one by "simply" changing the nodes rendered, and simplifying whatever methods don't exist on react native 🤔.
Might give it a try!

**Comment by jannomeister** [comment]
I agree with @juddey, I used the suggested projects by @alex-mironov but it's some kind of buggy :(

**Comment by parnekov** [comment]
Guys try this one. 
[](https://github.com/max-prokopenko/lowkey-react-native-mentions-input)

**Comment by walidvb** [comment]
@parnekov thanks for the tip. The licence is written permission only, though, and also, it doesn't seem that it addresses the bugs from the other projects, nor does it (currently) seem to allow other triggers(#hashtagsforexample), or a callback to async load the users.

Will keep an eye on it, though, thanks for sharing!

(and for a minimal example:)
https://snack.expo.io/@walidvb/lowkey-react-native-mentions-input
